manifests/pipelines/git-fetch-pipeline.yaml

```diff
@@ -108,8 +108,14 @@ spec:
   tasks:
   - name: fetch-model-git
```

Comment:
S3 version of the pipeline and GitOps pipeline should also have these, right?

Reply:
Yeap, I have that ready. Just haven't pushed it because of the buildah one.